packages/form-core/src/tests/FieldApi.spec.ts

```diff
@@ -107,6 +107,39 @@ describe('field api', () => {
     expect(field.getValue()).toStrictEqual(['one', 'other'])
   })
 
+  it.only('should run onChange validation when pushing an array fields value', async () => {
```

Comment:
I do this all the time too 😆 
```suggestion
  it('should run onChange validation when pushing an array fields value', async () => {
```

Reply:
Yeah an no matter how often you try to remember one always goes through, thanks for the review 😄